features/full_tests/batch_1/internal_error_reports.feature

```diff
@@ -33,7 +33,7 @@ Scenario: If an exception is thrown when sending errors/sessions then internal e
     # App data
     And the event "app.buildUUID" is not null
     And the event "app.id" equals "com.bugsnag.android.mazerunner"
-    And the event "app.releaseStage" equals "production"
+    And the event "app.releaseStage" matches "(production|development)"
```

Comment:
I'd be inclined to set `config.releaseStage` to something like 'mazerunner' in `prepareConfig()` so that all scenarios always have the same releaseStage. As the automatic setting of releaseStage is now covered by unit tests, this feels like the most consistent way of ensuring the releaseStage is populated correctly.